docs/provider-configuration.md

```diff
@@ -103,10 +103,6 @@ file.
   OpenStack control panels, this can be found at Access and Security > API
   Access > Credentials.
 * `password`: Refers to the password of a valid user set in keystone.
-* `tenant-id`: Used to specify the ID of the project where you want
```

Comment:
tenant-id /tenant-name one of it should be part of required parameter list rather than optional?

Reply:
tenant-id is an optional parameter, you may not specify it in a case you have tenant-name along with either domain-name or domain-id.
I think the best option would be to add information about this behavior in the "required" section.

Reply:
Thanks. sounds good